src/components/Fieldset.tsx

```diff
@@ -0,0 +1,98 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/components/lbh-fieldset/_fieldset.scss";
+
+/**
+ * The proptypes for {@link FieldsetLegend}.
+ */
+export interface FieldsetLegendProps {
+  id?: string;
+  className?: string;
+  isPageHeading?: boolean;
+  children?: React.ReactNode;
+}
+
+/**
+ * A component to set a legend for a {@link Fieldset} by passing a {@link FieldsetProps.legend} prop.
+ *
+ * ```ts
+ * // Creating a fieldset with a legend
+ * <Fieldset
+ *   id="fieldsetId"
+ *   className="fieldsetClass"
+ *   describedBy="fieldsetDescribedBy"
+ *   role="fieldsetRole"
+ *   legend={
+ *     <FieldsetLegend id="legendId" className="legendClass" isPageHeading>
+ *       children
+ *     </FieldsetLegend>
+ *   }
+ * >
+ *    children
+ * </Fieldset>
+ * ```
+ */
+export const FieldsetLegend: React.FunctionComponent<FieldsetLegendProps> = (
+  props: FieldsetLegendProps
+): JSX.Element => {
+  const { id, className, isPageHeading, children } = props;
+  return (
+    <legend id={id} className={classNames("govuk-fieldset__legend", className)}>
```

Comment:
```suggestion
    <legend id={id} className={classNames("govuk-fieldset__legend", "lbh-fieldset__legend", className)}>
```

Let's follow BEM even if there aren't any rules for the class (yet).